routes/api/verifyUser.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+
+const { verifyEmail, resendVerifyEmail } = require("../../controllers/auth");
+
+const { emailSchema } = require("../../schemas/users");
+
+const { validateBody } = require("../../decorators");
+
+const router = express.Router();
+
+router.get("/verify/:verificationToken", verifyEmail);
```

Comment:
verification token should be passed as bearer token in authorization in postman. It must be a middleware which will parse and decode that token, got user info from it and pass it next as req.user property